File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
a = int(input())
b = list(map(int, input().split()))
max =-1000000
min = 1000000
for i in range(0,a):
    if(max<b[i]):
        max = b[i]
    if(min>b[i]):
        min = b[i]
print(min, max)

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;
public class Main {
	private static Scanner scanner = new Scanner(System.in);
	public static void main(String[] args) {
		int x = scanner.nextInt();
		int[] a= new int[x];
		int max= -1000001, min= 10000001;
		for (int i=0;i<x;i++) {
			a[i]= scanner.nextInt();
		}

		for (int i=0; i<x; i++) {
			if(max <a[i]) {
				max = a[i];
			}
			if(min>a[i]) {
				min = a[i];
			}
		}
		System.out.print(min+ " "+max);
	}
}

